ansible/ci-playbooks/roles/ci-admin-user/tasks/unix.yml

```diff
@@ -10,6 +10,19 @@
     name: ci-admin
     comment: "CI System Administrator for Ansible use"
     group: ci-admin
-    password: "{{ ci_admin_password_hash }}"
     state: present
   become: yes
+
+- name: ci-admin-user (unix) | Set ci-admin user password
+  user:
+    name: ci-admin
+    password: "{{ ci_admin_password_hash }}"
+  become: yes
+  when: ansible_os_family != 'Darwin'
+
+- name: ci-admin-user (darwin) | Set ci-admin user password
+  user:
+    name: ci-admin
+    password: "{{ ci_admin_password }}"
+  become: yes
+  when: ansible_os_family == 'Darwin'
```

Comment:
Yes.  I think it could be done either way, and while we use separate files for some roles, I wonder if a role is a single task per platform, or a special case for a platform, it's worth putting the logic into a single file to keep things simple.